src/Item.php

```diff
@@ -151,15 +151,21 @@ public function get_item_tags($namespace, $tag)
     }
 
     /**
-     * Get the base URL value from the parent feed
-     *
-     * Uses `<xml:base>`
+     * Get the base URL value.
+     * Uses `<xml:base>`, or item link, or feed base URL.
      *
      * @param array $element
      * @return string
      */
     public function get_base($element = [])
     {
+        if (!empty($element['xml_base_explicit']) && isset($element['xml_base'])) {
+            return $element['xml_base'];
+        }
+        $link = $this->get_permalink();
+        if ($link != null) {
+            return $link;
+        }
         return $this->feed->get_base($element);
```

Comment:
[This method](https://github.com/simplepie/simplepie/blob/cce6ad227c742e54d1a1170087f3ef13bea06aa5/src/SimplePie.php#L2073-L2074) looks to me like it already checks the `xml:base` attribute on `$element`, so this PR makes the order as follows:

1. `xml:base`
2. Item’s permalink
3. `xml:base`
4. First `rel="alternate"` link in the feed
5. Feed URL

Reply:
Indeed, but I did not want to change the feeds own method, as it is called from several other places